src/app/resources/resources-add.component.ts

```diff
@@ -30,19 +30,23 @@ export class ResourcesAddComponent implements OnInit {
   media: string[];
   openWith: string[];
   resourceType: string[];
-  currentDate = new Date();
+  currentDate = new Date(); // might be forced to change date format using "toISOString().split('T')[0]"
```

Comment:
We were using timestamp in milliseconds to save so that we could format it on rendering as required. Can we use same over here as well?